doc/nrf/libraries/others/network_core_monitor.rst

```diff
@@ -43,13 +39,20 @@ During the startup of the network core, the reset bit is set and the cause of th
 This value is rewritten from the network core's ``RESET.RESETREAS`` register.
 For a detailed description of the bits in this register, see the `RESETREAS`_ section for nRF5340.
 
-The :c:func:`ncm_net_status_check` function returns the following error codes:
+The :c:func:`ncm_net_core_event_handler` function may be implemented by your application.
+A ``__weak`` implementation can be found in the :file:`nrf/subsys/net_core_monitor/app_core.c`.
+
+The following events are supported and are also listed in :file:`nrf/include/net_core_monitor.h`.
+
+* :c:macro:`NCM_RESET_EVENT`
```

Comment:
These names should be taken from `net_core_monitor.h`, right? They differ now

Reply:
You are right, these names should be taken from `net_core_monitor.h` file.